src/config.ts

```diff
@@ -1,15 +1,14 @@
 import dotenv from "dotenv";
 
+// Load environment variables
+dotenv.config();
```

Comment:
Yes, it is highly recommended to run the `dontenv.config()` as early as possible on entry into the application.  Ideally it would be in the `server.js` file but unfortunately that file uses an import from `@src/config.js`.  I'll try instead to use `import 'dotenv/config'` in the `server.js` file as the first import.